server/pages/index.js

```diff
@@ -143,7 +143,13 @@ function getDefaultContext( request ) {
 		isFluidWidth: !! config.isEnabled( 'fluid-width' ),
 		abTestHelper: !! config.isEnabled( 'dev/test-helper' ),
 		devDocsURL: '/devdocs',
-		store: createReduxStore( initialServerState )
+		store: createReduxStore( initialServerState ),
+		css: new Set(),
```

Comment:
`Set.prototype.forEach` respects the original insertion order, [the spec says](https://www.ecma-international.org/ecma-262/6.0/#sec-set.prototype.foreach). I'm not sure if the iterator (e.g., `set.entries()`) does, too -- the spec is not so clear about that.